tests/tests-core/src/main/java/com/gentics/mesh/test/TestDataProvider.java

```diff
@@ -570,8 +570,6 @@ public HibNode addFolder(HibNode rootNode, String englishName, String germanName
 		if (germanName != null) {
 			HibNodeFieldContainer germanContainer = contentDao.createFieldContainer(folderNode, german,
 				branch, userInfo.getUser());
-			// germanContainer.createString("displayName").setString(germanName);
-//			germanContainer.createString("teaser").setString(germanName);
 			germanContainer.createString("slug").setString(germanName);
```

Comment:
Display names are not created directly in the database anymore, but through the general purpose creation API, so this change is fair.